org.eclipse.xtext/src/org/eclipse/xtext/diagnostics/Diagnostic.java

```diff
@@ -54,4 +54,22 @@
 	 */
 	int getLength();
 
+    /**
+     * Returns the end line location of the issue within the source.
```

Comment:
Please also state that the end line is inclusive. I'd like to see a sentence about issues that span multiple lines, especially issues that have a region that ends with a newline.

Reply:
If the end line character is inclusive, then I don't know why test `org.eclipse.xtext.ide.tests.server.ValidatorTest.testDiagnosticAtEndOfLineIncludingNewline()` is green. It looks to me as if the jump to the start of the next line is also done in `org.eclipse.emf.ecore.resource.Resource.Diagnostic`.